src/main/java/ch/njol/skript/events/SimpleEvents.java

```diff
@@ -434,5 +436,23 @@
 							"		send \"Hello!\"")
 					.since("2.2-dev37");
 		}
+		if (Skript.classExists("com.destroystokyo.paper.event.player.PlayerJumpEvent")) {
+			Skript.registerEvent("Jump", SimpleEvent.class, PlayerJumpEvent.class, "[player] jump[ing]")
+					.description("Called whenever a player jumps",
+							"This event requires PaperSpigot")
+					.examples("on jump:",
+							"	event-player does not have permission \"jump\"",
+							"	cancel event")
+					.since("2.2-dev38");
+		}
+		if (Skript.classExists("org.bukkit.event.player.PlayerSwapHandItemsEvent")) {
+			Skript.registerEvent("Hand Item Swap", SimpleEvent.class, PlayerSwapHandItemsEvent.class, "swap[ping of] [(hand|held)] item[s]")
```

Comment:
what about:
```
hotbar (switch|swap) [to %itemtype%]
```
so you can do
```
on hotbar switch to diamond sword:
```

Reply:
Accept multiple item types, for `or` lists.